docs/api.md

```diff
@@ -373,6 +373,58 @@ Result:
 > - Clearing the local collection will mark the collection as never synchronized;
 > - Detailed API documentation for `Collection#clear()` is available [here](https://doc.esdoc.org/github.com/Kinto/kinto.js/class/src/collection.js~Collection.html#instance-method-clear).
 
+## Transactions
+
+Kinto.js exposes a concept of a "local transaction", which guarantees
+that a set of local operations will happen atomically -- that is, all
+or nothing.
+
+Note that as Kinto is still a distributed database, transactions
+aren't consistent. In other words, your "transaction" can still
+conflict with other operations on some other node.
```

Comment:
nitpick: I believe some abstraction could be removed from this paragraph. What does it mean for me frontend developer? Is Kinto really «distributed»? What is a «node»?